clustered/client/src/test/java/org/ehcache/clustered/client/UnSupportedCombinationsWithClusteredCacheTest.java

```diff
@@ -113,7 +109,7 @@ public void testClusteredCacheWithXA() throws Exception {
                   .withService(new XAStoreConfiguration("xaCache"))
                   .build()
           )
-          .build(true);
+          .build(true).close();
```

Comment:
A couple of things here ... is this test missing a `fail`?  Isn't the `StateTransitionException` thrown from `PersistentCacheManager.build` meaning `close` won't be called?  Would this one be better with a try-with-resources?

Reply:
Yes... fail was missing, added. Close should be called if the build erroneously executes cleanly. In the case that it fails (as expected) then it shouldn't since closing an uninitialized cache manager throws an ISE.